mautic/mautic
# Fix dashboard widget Sent email to contacts

| Q                                      | A
| -------------------------------------- | ---
| Branch?                                | "features"
| Bug fix?                               | yes
| New feature?                           | no
| Deprecations?                          | no
| BC breaks?                             | no
| Automated tests included?              | no
| Related user documentation PR URL      | 
| Related developer documentation PR URL | 
| Issue(s) addressed                     | 




#### Description:
The widget Sent email to contacts on dashboard does not display data. 
This PR fixed it



#### Steps to test this PR:
1. Load up [this PR](https://mautibox.com)
2. Create Sent email to contacts widget on dashboard
3. NO data displayed
4. After fix should display table width data
